Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+group :test do
+  gem 'rspec'
```

Comment:
2.14 is not 100% compatible with 3.x, for now and in our case is better if we do 2.x